### DISCUSSION TITLE
Is there a way to exclude attributes like `Cpp2IlInjected` for the dumped files?

### DISCUSSION THREAD

**Discussion by RaenonX** [icsharpcode/ILSpy#2562] [Help] [answered]
As title. Currently, using the console program in `ICSharpCode.Decompiler.Console` will have `Cpp2IlInjected`.

My main goal is just to compare the difference between 2 DLLs, so I don't want to have these attributes attached to affect the comparison results.

**Comment by RaenonX** [comment, marked as answer]
Oops it was generated from DLL, I was dumb and thinking that it's originated from `ILSpy`.

**Comment by siegfriedpammer** [comment]
There is no such functionality built in, but you can use ICSharpCode.Decompiler directly and add a simple transform that removes these attributes. See https://github.com/icsharpcode/ILSpy/blob/master/ICSharpCode.Decompiler.Tests/Helpers/RemoveCompilerAttribute.cs for an example.

Hope this helps.

**Reply by RaenonX** [reply]
Thanks! At last, I just found that I can suppress attributes from the dumper, so I won't have to play around with the console code
